### ISSUE TITLE
Bluetooth autoconnect and maxing out the volume

### ISSUE THREAD

**Issue by xtensa** [xtensa/PodEmu#12]
This is probably a bit out of scope. To have all fully automated (when getting into car), I need 2 more apps: Bluetooth Auto Connect and Bluetooth connect & play. The first one auto-connects to the AptX-Receiver as soon as HC-05 is connected (because my AptX-Rcv doesn't auto-connect unfortunately), and the second one maxes out Bluetooth Volume (very important, because it never ever sticks to max otherwise) and start playing Spotify. Would be really great if this could be implemented in PodEmu, so I don't need those 2 apps.

**Comment by xtensa** [comment]
Both problems described here are more related to automation and there are applications like Tasker that are supposed to deal with such tasks. 

What Apt-X receiver you are using? Most of them can be programmed to automatically max out the volume upon connection. Eg RN-52 gives this option. 

Regarding auto start playing favorite music app - this will be done.

**Comment by icstyle** [comment]
Yep that's true. That's way I'm using those 2 apps, which do the automation perfectly.

I'm using this bluetooth receiver: https://www.aliexpress.com/item/Ugreen-3-5mm-Bluetooth-Audio-Receiver-Universal-Car-Speaker-Music-Receiver-Adapter-with-Microphone-for-Headphone/32624733221.html
